pkg/liqoctl/install/validation.go

```diff
@@ -43,12 +43,14 @@ func (o *Options) validate(ctx context.Context) error {
 	o.Printer.Verbosef("Kubernetes API Server: %s\n", o.PodCIDR)
 
 	if err := o.validatePodCIDR(ctx); err != nil {
-		return fmt.Errorf("failed validating Pod CIDR %q: %w", o.PodCIDR, err)
+		return fmt.Errorf("failed validating Pod CIDR %q: %w. "+
+			"Try setting the correct PodCIDR by using the generic provider or by installing with Helm", o.PodCIDR, err)
```

Comment:
```suggestion
			"Try setting the correct Pod CIDR using the vanilla *liqoctl install* command, or installing Liqo with Helm", o.PodCIDR, err)
```

Definitely just a proposal (and same for the one below). I fear the term *generic provider* might generate a bit of confusion, since there is no *liqoctl install generic* command. Alternatively, we might take that route...